pulpcore/app/settings.py

```diff
@@ -223,7 +223,10 @@
     GLOBAL_ENV_FOR_DYNACONF='PULP',
     ENV_SWITCHER_FOR_DYNACONF='PULP_ENV',
     SETTINGS_MODULE_FOR_DYNACONF='/etc/pulp/settings.py',
-    INCLUDES_FOR_DYNACONF=['/etc/pulp/plugins/*'],
+    INCLUDES_FOR_DYNACONF=[
+        '{}.app.settings'.format(plugin_name) 
```

Comment:
Maybe it needs to be
```suggestion
        'pulp_{}.app.settings'.format(plugin_name) 
```

Reply:
And that may cause problems with plugin naming, like `pulp-certguard` uses `-` while `pulp_rpm` uses `_` I am really not sure..

Another solution is to try to extract the proper path from `entry_point` object https://github.com/pulp/pulpcore/blob/master/pulpcore/app/settings.py#L63

Reply:
I think what you have here is right. Based on my testing with pulp_file and pulp_ansible installed and this PR checked out, this is being set to `['pulp_file.app.settings', 'pulp_ansible.app.settings']`. Based on what you are saying is correct.